### PR TITLE
Test for spamaway

### LINKED ISSUE #4321: Integration Tests for Spam Away
>>I think we should have a test for this sequence with realistic parameters for spamaway, as it made me uncomfortable that it errored but the tests passed. 

see https://github.com/publiclab/plots2/pull/4209#issuecomment-447652453 and https://github.com/publiclab/plots2/pull/4209#issuecomment-447645413 for reference.

We need some tests to ensure that the spam away works fine on the localhost.
Let's build it @jywarren.
Refer #4209.

### PR DESCRIPTION
**Pull request by sssash18**
Fixes #4321  (<=== Add issue number here)
(part of issue)
Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by SidharthBansal** [comment]
@Uzay-G @VladimirMikulic what are your views?
Can we merge this?

File: test/integration/signup_flow_test.rb [hunk 1]
```diff
@@ -80,6 +80,25 @@ def setup
     assert response.body.include? 'errors prohibited this user from being saved'
     assert response.body.include? 'Email should look like an email address.'
   end
+  test 'spamaway text area not blank error message'
```
Comment (VladimirMikulic) [inline test/integration/signup_flow_test.rb]: There should be an empty line between the tests.

**Comment by VladimirMikulic** [comment]
@SidharthBansal we can merge this :)

**Comment by sssash18** [comment]
Thank you @SidharthBansal @VladimirMikulic for your approval.

**Comment by sssash18** [comment]
@VladimirMikulic @SidharthBansal Are their any more tests we need to add that I can help with?

**Comment by sssash18** [comment]
@VladimirMikulic Could you guide me about how to fix Travis here?

**Comment by SidharthBansal** [comment]
Ping any of us(@jywarren @SidharthBansal @VladimirMikulic @Uzay-G @cesswairimu ) if travis still fails

**Comment by VladimirMikulic** [comment]
Let me test it locally.

**Comment by SidharthBansal** [comment]
Sure

**Comment by VladimirMikulic** [comment]
Aha! I where is the problem, should have noticed that earlier :smile: 
You are missing the `do` keyword, once you add it your test will pass.

File: test/integration/signup_flow_test.rb (hunk 1 above)
Comment (VladimirMikulic) [reply test/integration/signup_flow_test.rb]:
```ruby
test 'something' do

end
```

**Comment by VladimirMikulic** [comment]
Check out my review comment.

**Comment by sssash18** [comment]
Sorry all of you for such a silly mistake

**Comment by VladimirMikulic** [comment]
Fingers :crossed_fingers:

**Comment by SidharthBansal** [comment]
No problem Suyash. If you will review my prs I can bet there will be 100
such errors.

**Comment by VladimirMikulic** [comment]
@sssash18 don't apologise! It happens to everyone.

**Comment by SidharthBansal** [comment]
We all are learning. Its important to continue learning despite of failures
and successes too

**Comment by sssash18** [comment]
@SidharthBansal I think now it's fine.

**Comment by VladimirMikulic** [comment]
Congratulations @sssash18 :rocket: :rocket: :rocket:

**Comment by cesswairimu** [comment]
This is awesome :tada: :tada:

**Comment by sssash18** [comment]
> Awesome!
> Just checking are you a GCI student?

No @SidharthBansal

**Pull request merged**